src/Network/MessagePack/Async/Client.hs

```diff
@@ -98,10 +107,13 @@ callRpc client funName args = do
 
 
 -- TODO: Receive Either MsgPack.Object MsgPack.Object
+-- | Replying RPC. This should be used in 'RequestHandler'.
 replyRpc :: Client -> MessageId -> MsgPack.Object -> IO ()
 replyRpc client mid result = do
-  let p = MsgPack.pack $ ResponseMessage mid (Right result)
-  clientSend client p
+  let response = ResponseMessage mid (Right result)
+  clientLog client "sent" response
```

Comment:
Strictly speaking, this line should be after `backendSend (clientBackend client) p` because it says "sent".

Reply:
Done.